src/graphql/admin/root/mutation/account-update-level.ts

```diff
@@ -22,6 +23,7 @@ const AccountUpdateLevelMutation = GT.Field({
     input: { type: GT.NonNull(AccountUpdateLevelInput) },
   },
   resolve: async (_, args) => {
+    // FIXME: should no longer be uid
```

Comment:
What should it be then?
I don't see the value in this comment - saying what should _not_ be the case doesn't really help in indicating what _should_ be done. As a reader I don't get any information from this other than being confused. The questions in my head on reading this is:
- so the current code is wrong?
- does that means its broken?
- what is broken about it?
- if its broken why don't we fix it?
- why do tests pass?
- what about clients using this field?
... etc.

Maybe I'm lacking context here - I'm just giving my train of thought when I read this comment.

Reply:
it should not be `uid`/`user id` but `aid`/`account id`. comment updated

Reply:
it's part of the refactoring to dissociated user/account/wallet to 3 different entities.